index.html

```diff
@@ -1,179 +1,254 @@
-                <li>Ruby</li>
+              <li class="skillset">Codekit</li>
+              <li class="skillset">Github</li>
+              <li class="skillset">Codepen</li>
+              <li class="skillset">Gitlab</li>
+              <li class="skillset">Terminal</li>
             </ul>
-            <button type="button">See Project</button>
-        </div>
+          </li>
+        </ul>
+      </div>
     </section>
-    <section class="about-me">
-        <h1>About me</h1>
-        <p>Hello, I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-        <button type="button">Get My Resume</button>
-        <hr>
-        <div>
-            <ul class="skills">
-                <li class="skills-list"><img src="assets\Rectangle 55.png" alt=""> Language
-                    <ul>
-                        <li class="skillset">Javascript</li>
-                        <li class="skillset">Ruby</li>
-                        <li class="skillset">Html</li>
-                        <li class="skillset">Css</li>
-                    </ul>
-                </li>
-                <li class="skills-list"><img src="assets\Rectangle 56.png" alt=""> Frameworks
-                    <ul>
-                        <li class="skillset">Bootstrap</li>
-                        <li class="skillset">Ruby on Rails</li>
-                        <li class="skillset">RSPec</li>
-                        <li class="skillset">CapyBara</li>
-                        <li class="skillset">Selenium</li>
-                    </ul>
-                </li>
-                <li class="skills-list"><img src="assets\Rectangle 57.png" alt=""> Skills
-                    <ul>
-                        <li class="skillset">Codekit</li>
-                        <li class="skillset">Github</li>
-                        <li class="skillset">Codepen</li>
-                        <li class="skillset">Gitlab</li>
-                        <li class="skillset">Terminal</li>
-                    </ul>
-                </li>
-            </ul>
-        </div>
+    <section class="form">
+      <h1 class="form-header">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h1>
+      <form action="https://formspree.io/f/xvoyvqjy" method="post" id="formid">
+        <input
+          class="form-input"
+          type="text"
+          placeholder="Full name"
+          name="Full name"
+          maxlength="30"
+          required
+        />
+        <input
+          class="form-input"
+          type="email"
+          placeholder="Email address"
+          name="Email address"
+          required
+        />
+        <textarea
+          name="Message"
+          id="textarea"
+          cols="50"
+          rows="10"
+          placeholder="Enter text here"
```

Comment:
Kindly add the maxlength property to your form as required for HTML validation in the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md)
<img width="1518" alt="Screenshot 2022-11-01 at 14 57 53" src="https://user-images.githubusercontent.com/48546992/199429694-d079462e-6c5b-4dc8-a7ea-26ee0366de36.png">